developer/src/kmcmplib/src/CompilerInterfaces.cpp

```diff
@@ -162,6 +129,7 @@ EMSCRIPTEN_BINDINGS(compiler_interface) {
     ;
 
   emscripten::function("kmcmp_compile", &kmcmp_compile);
+  emscripten::function("kmcmp_parseUnicodeSet", &kmcmp_parseUnicodeSet);
```

Comment:
Is this the 'replace' part?  The bit that lets `emscripten` take over binding?

Reply:
Yes